doc/zmq.md

```diff
@@ -46,11 +46,10 @@ operation.
 
 ## Enabling
 
-By default, the ZeroMQ feature is automatically compiled in if the
-necessary prerequisites are found.  To disable, use -DWITH_ZMQ=OFF
-during the *configure* step of building bitcoind:
+By default, the ZeroMQ feature is not automatically compiled.
+To enable, use `-DWITH_ZMQ=ON` when building:
```

Comment:
pedantic nit: The technically correct term for the first `cmake` invocation is ["generating a project buildsystem"](https://cmake.org/cmake/help/latest/manual/cmake.1.html#generate-a-project-buildsystem). The actual "building" is performed by the generated build system during the following step. Elsewhere in our documentation, we use phrases such as "configuring Bitcoin Core" or "configuring the build system".

Reply:
Yeah, you're right. I was on the fence about naming (since the actual build is done with `cmake --build build -jN`. Better to fix it now, so I pushed a change.